src/store/modules/ADempiere/pointOfSales/payments/state.js

```diff
@@ -30,5 +30,10 @@ export default {
   currency: [],
   convertion: {},
   fieldCurrency: {},
-  convertionRate: []
+  convertionRate: [],
+  dialogoInvoce: {
```

Comment:
please change state name to `dialogInvoice` (without the `o` at the end of the dialog)